.eslintrc.cjs

```diff
@@ -39,10 +39,25 @@ module.exports = {
     // Typescript does this for us
     'relay/function-required-argument': 'off',
     // Typescript does this for us
-    'relay/hook-required-argument': 'off'
+    'relay/hook-required-argument': 'off',
   },
 
   overrides: [
+    // This is temporary while we have two versions of the collection editor
+    // in production at once. We want to ensure that the newer version is
+    // never importing from the older version.
+    {
+      files: ['src/components/GalleryEditor/**/*'],
+      rules: {
+        'no-restricted-imports': [
+          'error',
+          {
+            patterns: ['~/components/ManageGallery/OrganizeCollection/*'],
+          },
+        ],
+      },
+    },
+
```

Comment:
really dig this for the migration period

Reply:
this is awesome